docs/manuals/core/config/global-config.md

```diff
@@ -13,7 +13,18 @@ Here are the (optional) configurable properties:
   feature only on specific environments. That is why the default value points to an environment variable.
 - _**read_entity_retry**_: The integer number of retry when reading an entity after a failed attempt (in case of
   concurrent access). The default value is 0.
-- _**properties**_: The dictionary of additional properties.
+- _**repository_type**_: The type of storage that will be used to hold Taipy entities. Available options are:
+  `filesystem`, `sql` and `mongo`. If no repository type is informed, the filesystem will be used as the default
+  repository.
+- _**repository_properties**_: A dictionary of properties that will be used to instantiate the chosen repository.
+  Only required if the chosen repository is `sql` or `mongo`.</br>
+  If _**repository_type**_ is set to `filesystem`, Taipy uses _**root_folder**_ and _**storage_folder**_ to store
+  entities. In this case, _**repository_properties**_ attribute is not required.</br>
+  If _**repository_type**_ is set to `sql`, the _**db_location**_ attribute is required as the path of a sqlite3
+  database file. Please refer to [SQL storage section](global-config.md#sql-storage-for-taipy-entities).</br>
+  If _**repository_type**_ is set to `mongo`, the possible properties are _**mongodb_hostname**_, _**mongodb_user**_,
+  _**mongodb_password**_, _**mongodb_port**_, _**application_db**_, _**properties**_. Please refer to
```

Comment:
@joaoandre-avaiga I added some more details here.

Reply:
Looks good to me. Thank you.